doc/start_docs.py

```
@@ -0,0 +1,30 @@
+    """Call the main start_docs.py script with prs-rs specific parameters."""
+    # Get the directory where this script is located
+    script_dir = Path(__file__).parent
+    reloaded_script = script_dir / "docs" / "Reloaded" / "start_docs.py"
+    
+    if not reloaded_script.exists():
+        print(f"Error: Could not find {reloaded_script}")
+        sys.exit(1)
+    
+    # Run the main script with parameters for prs-rs
+    print(f"Running prs-rs documentation setup from: {reloaded_script}")
+    subprocess.run([
```

Ruff (0.14.1)

`subprocess` call: check for execution of untrusted input

(S603)